views/Account.react.js

```diff
@@ -123,15 +124,13 @@ class AccountPage extends Component {
     user['icon'] = this.state.icon
 
     return window
-      .fetch(`${basePath}/api/v1/users/${uid}`, {
-        method: 'POST',
+      .fetch(apiRoutes.users.user(uid), {
+        method: 'PUT',
```

Comment:
I think technically this should be a `PATCH`.

Reply:
[f11e811](https://github.com/AMP-SCZ/dpdash/pull/305/commits/f11e811a37bb7df8cc158a8d0e061615e8ec535d)